cran-comments.md

```diff
@@ -1,12 +1,22 @@
-## v. 1.1.2
-* Fixed bug when tokenizing empty string.
+## v. 1.2.1 (Resubmission)
```

Comment:
I think "Resubmission" implies submitting after they rejected. Have we worded it that way before?